README.md

```diff
@@ -143,13 +146,13 @@ There may also be API packages for experimental signals in the experimental dire
 All stable packages are released with the same version, and all experimental packages are released with the same version.
 The below table describes which versions of each set of packages are expected to work together.
 
-| Stable Packages                                                 | Experimental Packages |
-|-----------------------------------------------------------------|-----------------------|
-| 1.21.x                                                          | 0.48.x                |
-| 1.20.x                                                          | 0.47.x                |
-| 1.19.x                                                          | 0.46.x                |
-| 1.18.x                                                          | 0.45.x                |
-| 1.17.x                                                          | 0.44.x                |
+| Stable Packages | Experimental Packages |
+| --------------- | --------------------- |
+| 1.21.x          | 0.48.x                |
+| 1.20.x          | 0.47.x                |
+| 1.19.x          | 0.46.x                |
+| 1.18.x          | 0.45.x                |
+| 1.17.x          | 0.44.x                |
```

Comment:
The table cleanup is cool. This makes me realize this section is way out of date and a maintenance pain.

The "semconv" package is now a new category for the "OpenTelemetry is released as a set of distinct packages in 3 categories:" sentence at the start of this section.

Anyway... not for this PR.

Reply:
Oh actually I didn't mean to reformat anything in this doc. It must have auto-formatted when I added the first section and saved.